src/clj/web/api.clj

```diff
@@ -29,6 +30,10 @@
            (POST "/logout" [] auth/logout-handler)
            (GET "/check/:username" [] auth/check-username-handler)
            (PUT "/profile" [] auth/update-profile-handler)
+           (GET "/user" [] data/get-user)
```

Comment:
Please follow the "xx-handler" naming scheme of the other route-handling functions.

Reply:
Yep - ok - though I think I can delete this route once the web-socket sends the stats update content itself.